internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -87,7 +87,11 @@ func (s *BackendService) Template() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	sidecars, err := convertSidecar(s.manifest.Sidecars)
+	_, err = s.manifest.ExposedPorts()
```

Comment:
Can `ExposedPorts` be a private function in of `manifest`?

Reply:
Made required changes as per the offline discussion.